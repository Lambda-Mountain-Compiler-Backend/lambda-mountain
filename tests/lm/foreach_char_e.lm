
main := (
   foreach-char (hello world) ((print-s $_) (print-s '_))
);
