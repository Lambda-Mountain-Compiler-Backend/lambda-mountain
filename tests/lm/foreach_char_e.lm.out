h_e_l_l_o_w_o_r_l_d_
